README.md

```diff
@@ -55,10 +55,30 @@ You likely won't need to run `plasma-harness` in production.
 ### 1. Checkout, install deps, and build:
 
 ```bash
+# install go from (https://golang.org/dl/)
+go env GOPATH # should be ~/go
```

Comment:
Can you remove these guys (up to line 76), and instead roll them up into a paragraph above the code block? Reason is so that we can make it clear which steps are Plasma-specific, and which aren't.

Reply:
Should be addressed now